data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+	on complete
+		payment 100000
+		dialog
+			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
+		"reputation: Hai" += 10
+
+
+
+phrase "Essential Unfettered Mission Fail"
+	word
+		`You have failed an essential Unfettered mission. You may want to reload.`
+
+
+
+mission "Unfettered: Neutral With The Hai"
+	name `Get forgiven by the Hai`
+	description `The Unfettered have asked you to go negotiate with the Hai to stop your conflict with them. Go to <destination>.`
+	source
+		attributes "unfettered"
+	destination "Mirrorlake"
+	clearance `You speak with the local authorities and they agree to meet with you.`
+	to offer
+		has "reputation: Hai" < 0
+		has "Unfettered Jump Drive 3: offered"
+	on offer
+		conversation
+			`On your promenade you find an Unfettered elder who chastises you as a father would to a misbehaving child, "What have you done to anger our brethren?" You begin to answer, but he immediately interrupts you, "Never mind, it matters not. What matters is that you settle this petty dispute you have with the Hai government right now! Or, in the future, cooperation with us will become difficult."`
+			choice
+				`	(Accept.)`
+					goto agree
+				`	"You have no place to tell me what to do."`
+				`	"I can manage your weak brethren, bother me not with this again."`
+					decline
+			`	He looks at you gravely, "Your loss. Do not come here crying when the Hai are causing you difficulties."`
+			choice
+				`	"Wait, I have changed my mind!"`
+				`	(Go.)`
+					decline
+			`	He makes a show of thinking it over before responding, "Well, I guess I can forgive your insolence, this time. But listen to what I am about to say."`
+			label agree
+			`	"Go to their home system and make your request, the Hai government despises conflict, so they will most likely grant it. They won't let you land on their homeworld given the circumstances, though, so go to Mirrorlake instead."`
+				accept
+	on complete
+		conversation
+			`You find the Hai leaders that agreed to meet you and express your request to settle the disputes between you. They consider this amongst themselves before responding, "Tell us, <last>, captain of the <ship>, why should we grant you such a request?"`
+			choice
+				`	"Because otherwise you get shot!" (Draw your weapon)`
+					goto death
+				`	"I know I have done stupid things in the past, but I have learned my error. I promise not to make such mistakes again."`
+				`	"I can continue bringing food to the Unfettered for you, so they do not starve to death."`
+			`	They seem willing to give you a second chance. "You have convinced us, but we will not grant you this pardon twice."`
```

Comment:
These choices definitely need to have meaningful outcomes, and probably be revisited.
As it stands if I was one of the Elders I'd laugh in your face and have you arrested with either of these answers.

Reply:
What you probably need is a mission or discussion before this to lay more groundwork so you can bring them a justified reason.

Reply:
The player is not supposed to see this mission, except if he fucks up, and I can only think of 2 missions where you help the Hai, I really meant this as "oh you though you could be at war with the Hai when siding with UHai? well no worries dudes, now rep is reset but don't do it again"

Reply:
I developped in furhter, do tell me what you think